pvlib/modelchain.py

```diff
@@ -924,6 +925,7 @@ def run_model(self, weather, times=None):
         -------
         self
 
+
```

Comment:
Maybe move this up to the Extended summary position above the Parameters

Reply:
Changed -- I liked the way `prepare_inputs` lists these in the `Notes` section, so I've changed the other two method docstrings to do the same.